packages/immutable-rootfs/30cos-immutable-rootfs/module-setup.sh

```diff
@@ -22,7 +22,6 @@ install() {
     declare moddir=${moddir}
     declare systemdutildir=${systemdutildir}
     declare systemdsystemunitdir=${systemdsystemunitdir}
-    declare initdir="${initdir}"
```

Comment:
THANK YOU. This kept printing an error if Im not mistaken and was fedup of seeing it

Reply:
Yes indeed, you are right, this was a pretty annoying warning